readme.md

```diff
@@ -0,0 +1,58 @@
+Custom Yup Locales
+====================
+
+[![npm version](https://badge.fury.io/js/%40aboutbits%2Fyup-locales.svg)](https://badge.fury.io/js/%40aboutbits%2Fyup-locales)
+
+This package includes simplified translations for yup error messages. The main difference lies in the exclusion of the field path.
+
+## Table of content
+
+- [Usage](#usage)
+- [Build & Publish](#build--publish)
+- [Information](#information)
+
+## Usage
+
+First, you have to install the package:
+
+```bash
+npm install @aboutbits/yup-locales
+```
+
+Second, you add the custom language packages to your project.
+
+```js
+import { setLocale } from 'yup'
+import { en, it, de } from '@aboutbits/yup-locales'
+import { LocaleObject } from 'yup/lib/locale'
+import { Language } from '../components/support/types'
```

Comment:
Are these two required here in the example?

Reply:
nope, thanks :)